light/proxy/routes.go

```diff
@@ -141,40 +133,6 @@ func makeCommitFunc(c *lrpc.Client) rpcCommitFunc {
 	}
 }
 
-type rpcDataCommitmentFunc func(
-	ctx *rpctypes.Context,
-	start uint64,
-	end uint64,
-) (*ctypes.ResultDataCommitment, error)
-
-type rpcDataRootInclusionProofFunc func(
-	ctx *rpctypes.Context,
-	height uint64,
-	start uint64,
-	end uint64,
-) (*ctypes.ResultDataRootInclusionProof, error)
-
-func makeDataCommitmentFunc(c *lrpc.Client) rpcDataCommitmentFunc {
-	return func(
-		ctx *rpctypes.Context,
-		start uint64,
-		end uint64,
-	) (*ctypes.ResultDataCommitment, error) {
-		return c.DataCommitment(ctx.Context(), start, end)
-	}
-}
-
-func makeDataRootInclusionProofFunc(c *lrpc.Client) rpcDataRootInclusionProofFunc {
-	return func(
-		ctx *rpctypes.Context,
-		height uint64,
-		start uint64,
-		end uint64,
-	) (*ctypes.ResultDataRootInclusionProof, error) {
-		return c.DataRootInclusionProof(ctx.Context(), height, start, end)
-	}
-}
```

Comment:
Why are we removing this?

Reply:
they're unused atm

Reply:
Do we want to support them in the future?

Reply:
I lean towards deleting unused code. In the future if the code needs to be reintroduced, we can find it in git history.